homework-001/Hello.js

```diff
@@ -0,0 +1,300 @@
+// **************************************************************************
+
+// Calvin Hui | Assignment One
+
+// **************************************************************************
+
+// Variables
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill'],
+    }
+};
+
+// This counter is used to number the board names
+let counter = 1;
```

Comment:
variables that are only used within a function should be defined in that function.